docs/access/macos.rst

```diff
@@ -0,0 +1,12 @@
+.. _access_mac:
+
+MacOS
+#####
+
+To start the **Kolibri** server on macOS, just double-click the Kolibri app from the **Launchpad**. 
```

Comment:
Until we get our app signed, we'll want to link to this documentation:

https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac